openedx/core/djangoapps/theming/core.py

```diff
@@ -47,7 +47,7 @@ def comprehensive_theme_changes(theme_dir):
 
     favicon = component_dir / "static" / "images" / "favicon.ico"
     if favicon.isfile():
-        changes['settings']['FAVICON_PATH'] = str(favicon)
+        changes['settings']['FAVICON_PATH'] = "images/favicon.ico"
```

Comment:
@ziafazal previously `str(favicon)` returned complete path i.e. `/edx/app/edxapp/edx-platform/themes/stanford-style/lms/static/images/favicon.ico`, which returned a 404 on request and edx favicon was being used instead.
that is why updated it to `"images/favicon.ico"` so that correct favicon path is used. But I do also need confirmation on this as whether or not favicon was working previously with comprehensive theming.